### PR TITLE
Release 1.6.2

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Release 1.6.2

### CODE REVIEW

File: ovirt-engine-metrics.spec.in [hunk 1]
```diff
@@ -70,6 +70,10 @@ rm -rf %{buildroot}
 %doc README.md
 
 %changelog
+* Tue Nov 28 2023 Sandro Bonazzola <sandro.bonazzola@gmail.com> - 1.6.2-1
```
Comment (mwperina) [inline ovirt-engine-metrics.spec.in:73]: Could you please add following change?
```
-  Fix comparison logic in ovirt_initial_validations Ansible role https://github.com/oVirt/ovirt-engine/issues/869
```

Comment (mwperina) [reply ovirt-engine-metrics.spec.in:73]: It was the only important change since 1.6.1

Comment (sandrobonazzola) [reply ovirt-engine-metrics.spec.in:73]: done

**Review by mwperina — APPROVED** [review]
LGTM

**Pull request merged**